backend/src/main/resources/schema.sql

```diff
@@ -0,0 +1,33 @@
+CREATE TABLE STAFF (
```

Comment:
Sorry I havent had time to look at this until now.

So i mentioned in the call that I'm happy for you guys to move on doing it like this if you have an idea of how it works. However, I'm more familiar using classes and JPA repositories with a Sping project.

Here you've outlined the tables that we need. What you should also do in this PR is create a java class for each of these tables you're creating.

Then, rather than using a db maipulation class in the other PR where you're manually writing the SQL, we can create a repository class, which comes with default methods such as `staffRepository.delete(id)`, or `staffRepository.add(newStaff)`

For example, on the magic app we have a sql table called UserBalance, which looks like this:
![Screen Shot 2020-04-14 at 13 12 56](https://user-images.githubusercontent.com/39953553/79224457-03d69000-7e53-11ea-9bcc-5b0dfff1ae44.png)


And to store this, we've created a repository in the same directory which looks like this:
![Screen Shot 2020-04-14 at 13 13 11](https://user-images.githubusercontent.com/39953553/79224466-089b4400-7e53-11ea-8e37-177f31c828a6.png)


Then in all you classes where you need to use this repository (eg from an endpoint) you can use `@Autowired StaffRespository staffRepository` at the top of the class and then just call those staffRepository.getAll() or staffRepository.save(object) methods which come predefined. If you need custom methods for the repo you can define those in the Repository class too